crates/core/tedge_agent/src/software_manager/tests.rs

```diff
@@ -64,6 +65,11 @@ async fn test_new_software_update_operation() -> Result<(), DynError> {
 
     let mut converter_box = spawn_software_manager(&temp_dir).await?;
 
+    // software cmd metadata internal message
+    converter_box
+        .assert_received([SoftwareCommandMetadata { types: vec![] }])
```

Comment:
It would be better to update the test so there is at least one software type.

Reply:
Applied in https://github.com/thin-edge/thin-edge.io/pull/2730/commits/4977f01edaaec5b8e3ecc83ad590feeee16385de